cfme/tests/middleware/test_middleware_provider.py

```diff
@@ -54,7 +54,10 @@ def test_password_mismatch_validation(provider, soft_assert):
         verify_secret=fauxfactory.gen_alphanumeric(6)
     )
     prov.endpoints = endpoints
-    with error.expected('Credential validation was successful'):
+    message = 'Credential validation was successful'
+    if current_version() >= '5.9':
```

Comment:
**Optional**
VERY optional, more a FYI

This isn't a new import, but in the future, it is a better approach to take appliance as a fixture for your test, and use its version attribute instead of the current_version() method.

Reply:
I see, I did not find any other better examples implemented, so made in the old way